docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md

```diff
@@ -281,137 +146,318 @@ export const httpLoaderFactory = (httpClient: HttpClient) => {
-
-  ngOnInit(): void {
-    this.eventService
-      .registerForEvents()
-      .pipe(filter((notification) => notification.type === EventTypes.NewAuthorizationResult))
-      .subscribe((result: OidcClientNotification<AuthorizationResult>) => {
-        console.log('NewAuthorizationResult', result);
-        console.log('isAuthenticated', result.value.isAuthenticated);
-      });
-  }
+```ts
+export interface OpenIdConfiguration {
+  autoUserInfo?: boolean;
+  // ...
 }
 ```
 
-### `logoff` expects object now
+### Custom Params to pass to requests have been renamed
 
-The parameter to be passed into the `logoff` method is now an object with the properties `urlHandler` and `customParams` which provides you to pass custom parameters to the logoff url.
+Old:
 
-#### Old
+```ts
+export interface OpenIdConfiguration {
+  customParams?: { [key: string]: string | number | boolean };
+  customParamsRefreshToken?: { [key: string]: string | number | boolean };
+  customParamsEndSession?: { [key: string]: string | number | boolean };
+  customTokenParams?: { [key: string]: string | number | boolean };
+  // ...
+}
+```
 
-```typescript
-logoff(urlHandler?: (url: string) => any) {
-  return this.logoffRevocationService.logoff(urlHandler);
+New:
+
+```ts
+export interface OpenIdConfiguration {
+  customParamsAuthRequest?: { [key: string]: string | number | boolean };
+  customParamsRefreshTokenRequest?: { [key: string]: string | number | boolean };
+  customParamsEndSessionRequest?: { [key: string]: string | number | boolean };
+  customParamsCodeRequest?: { [key: string]: string | number | boolean };
+  // ...
+}
+```
+
+## Service `OidcSecurityService`
+
+### Getter for `configuration` and object `PublicConfiguration` replaced by method
+
+The getter for the active configuration was removed as well as the object `PublicConfiguration`. As the `AuthWellKnownEndpoints` are now part of the config the property was replaced with methods to get the configuration directly. Please read below how to get the current used configuration.
+
+Old:
+
+```ts
+export class OidcSecurityService {
+  get configuration(): PublicConfiguration {}
 }
 ```
 
+used with
+
+```ts
+const configuration = this.oidcSecurityService.configuration;
+```
+
+New:
+
+```ts
+getConfigurations(): OpenIdConfiguration[] { }
+
+
+getConfiguration(configId?: string): OpenIdConfiguration { }
+```
+
+used with
+
+```ts
+const configuration = this.oidcSecurityService.getConfiguration();
+// or
+const configuration = this.oidcSecurityService.getConfiguration('configId');
+```
+
+### `getToken()` renamed to `getAccessToken()`
+
+Old
+
+```ts
+const accessToken = this.oidcSecurityService.getToken();
+```
+
+New
+
+```ts
+const accessToken = this.oidcSecurityService.getAccessToken();
+```
+
+### `checkAuth()` returning `LoginResponse` instead of boolean
+
+The `checkAuth()` method is now not only returning if you are authenticated or not but instead a complete object containing:

```

Comment:
Reword for clarity

The `checkAuth()` method no longer returns a boolean value denoting the authentication status. Instead, an object is returned containing: